pubsub/iterator.go

```diff
@@ -76,7 +76,13 @@ type messageIterator struct {
 func newMessageIterator(subc *vkit.SubscriberClient, subName string, maxExtensionPeriod *time.Duration, po *pullOptions) *messageIterator {
 	var ps *pullStream
 	if !po.synchronous {
-		ps = newPullStream(context.Background(), subc.StreamingPull, subName, po.maxOutstandingMessages, po.maxOutstandingBytes)
+		maxMessages = po.maxOutstandingMessages
+		maxBytes = po.maxOutstandingBytes
+		if po.UseLegacyFlowControl {
```

Comment:
CI is currently failing because this needs to be lower case

```suggestion
		if po.useLegacyFlowControl {
```